lib/tasks/panopticon.rake

```diff
@@ -85,4 +85,22 @@ namespace :panopticon do
       end
     end
   end
+
+  desc "Re-register all content with Panopticon"
+  task :re_register_all_content => :environment do
+    logger = GdsApi::Base.logger = Logger.new(STDERR).tap { |l| l.level = Logger::INFO }
+
+    published_editions = Edition.all
```

Comment:
The default scope for Editions is [anything not in a "deleted" state](https://github.com/alphagov/whitehall/blob/0e86b571450cd381c8ca62a6e1377fedbbcf0cf2/app/models/edition/workflow.rb#L21). This is probably what you want anyway so that withdrawn etc artefacts are re-published, but:

1. The variable name is a bit misleading.
2. You will want to de-duplicate the editions by document, else you'll end up publishing each artefact multiple times if the document has multiple editions.